website/source/guides/security/vault-pki-integration.html.md

```diff
@@ -0,0 +1,550 @@
+```
+
+### Step 7: Create a Policy to Access the Role Endpoint
+
+Recall from [Step 1](#step-1-initialize-vault-server) that we generated a root
+token that we used to log in to Vault. Although we could use that token in our
+next steps to generate our TLS certs, the recommended security approach is to
+create a new token based on a specific policy with limited privileges.
+
+Create a policy file named `tls-policy.hcl` and provide it the following
+contents:
+
+```
+path "pki_int/issue/nomad-cluster" {
+  capabilities = ["update"]
+}
+```
+Note that we have are specifying the `update` [capability][capability] on the
+path `pki_int/issue/nomad-cluster`. All other privileges will be denied. You can
+read more about Vault policies [here][policies].
+
+Write the policy we just created into Vault:
+
+```
+$ vault policy write tls-policy tls-policy.hcl
+Success! Uploaded policy: tls-policy
+```
+
+### Step 8: Generate a Token based on `tls-policy`
+
+Create a token based on `tls-policy` with the following command:
+
+```
+$ vault token create -policy="tls-policy" -ttl=24h
+```
+
+If the command is successful, you will see output similar to the following:
+
+```shell
+Key                  Value
+---                  -----
+token                s.xafiYzh7MCMotHLu2d35hepR
+token_accessor       9vj7q5nnF53JAcTyxvccpAZK
+token_duration       24h
+token_renewable      true
+token_policies       ["default" "tls-policy"]
+identity_policies    []
+policies             ["default" "tls-policy"]
+```
+
+Make a note of this token as you will need it in the upcoming steps.
+
+### Step 9: Configure Consul Template on All Nodes
+
+If you are using the AWS environment provided in this guide, you already have
+[Consul Template][consul-template-github] installed on all nodes. If you are
+using your own environment, please make sure Consul Template is installed. You
+can download it [here][ct-download].
+
+Provide the token you created in [Step
+8](#step-8-generate-a-token-based-on-tls-policy) to the Consul Template
+configuration file located at `/etc/consul-template.d/consul-template.hcl`. You
+will also need to specify the [template stanza][ct-template-stanza] so you can
+render each of the following on your nodes at the specified location from the
+provided templates (the actual templates will be provided in the next step):
+
+* Node certificate
+* Node private key
+* CA public certificate
+
+We will also specify the template stanza to create certs and keys for the Nomad
+CLI (which defaults to HTTP but will need to use HTTPS once once TLS is enabled
+in our cluster).
+
+Your `consul-template.hcl` configuration file should look similar to the
+following (you will need to provide this to each node in the cluster):
+
+```
+vault {
+  address      = "http://active.vault.service.consul:8200"
+  token        = "s.xafiYzh7MCMotHLu2d35hepR"
+  grace        = "1s"
+  unwrap_token = false
+  renew_token  = true
+}
+
+syslog {
+  enabled  = true
+  facility = "LOCAL5"
+}
+
```

Comment:
Minor suggestion - perhaps we can add comments in-line in this file to help people distinguish and walk through this file even if copypasta?  E.g this section could be prefaced with ##"Node Templates`

Reply:
Added some comments to this config. Let me know if you feel I over did it.